lib/orogen/gen/typekit.rb

```diff
@@ -1212,6 +1212,7 @@ def load(file, add = true, user_options = Hash.new)
                 include_path = include_dirs.map { |d| Pathname.new(d) }
                 inc = resolve_full_include_path_to_relative(file, include_path)
                 included_files << inc
+                user_options[:rawflags] = self.used_libraries.map do |lib| lib.raw_cflags end.flatten.uniq
```

Comment:
For consistency in the style used in orogen (and the general ruby style guide), either use a multi-line `do...end` or single-line `{ ... }`, e.g.

~~~
map do
end.flatten.uniq 
{ |lib| ... }.flatten.uniq
~~~